quickjs.h

```diff
@@ -294,6 +294,7 @@ JS_EXTERN JSRuntime *JS_NewRuntime(void);
 /* info lifetime must exceed that of rt */
 JS_EXTERN void JS_SetRuntimeInfo(JSRuntime *rt, const char *info);
 JS_EXTERN void JS_SetMemoryLimit(JSRuntime *rt, size_t limit);
+JS_EXTERN void JS_SetDumpFlags(JSRuntime *rt, unsigned flags);
```

Comment:
If we are making the flags configurable, how about making the stream where we dump the output configurable too?

Reply:
Good point, I have another patch in the pipeline where the output stream is passed and used for the dump code, along with a custom version of printf to output `JSAtom`, `JSString` and JSValue` directly.
I made separate patches to keep them manageable.

Reply:
Sounds good!